crates/polars-plan/src/dsl/function_expr/datetime.rs

```diff
@@ -105,6 +113,14 @@ impl TemporalFunction {
             MonthStart => mapper.with_same_dtype(),
             #[cfg(feature = "date_offset")]
             MonthEnd => mapper.with_same_dtype(),
+            #[cfg(feature = "date_offset")]
```

Comment:
FYI for brevity you can split this with the `|` symbol:

```rust
#[cfg(feature = "date_offset")]
MonthStart | MonthEnd | QuarterStart | QuarterEnd | YearStart | YearEnd => mapper.with_same_dtype(),
```

Reply:
nice one, thank you!